src/styles/01-atoms/_images/image/_image.scss

```diff
@@ -1,18 +1,30 @@
 @mixin image {
-  display: block;
-  height: auto;
-  max-width: 100%;
+    display: block;
+    height: auto;
+    max-width: 100%;
 }
 
 img,
 picture {
-  @include image;
+    @include image;
 }
 
 figure {
-  margin: 0; //override normalize
+    margin: 0; //override normalize
 }
 
 .figure__caption {
-  font-style: italic;
+    font-style: italic;
+}
+
+.annotation__image {
+    p {
+        font-style: italic;
+    }
+    p:nth-child(odd) {
```

Comment:
These overrides are a little odd. I'd recommend going forward with a `span` or `div` for the credit and caption instead, and then you won't have to override the `p` styling.

Also as an FYI we have helpers to restrict spacing to a strict taxonomy. You can find all of those defined in the "space/" folder under `styles/`.

`@include space-stack-none`

Reply:
Thanks for pointing out the spacing variables. I have updated to use `@include space-stack-none`